.travis.yml

```diff
@@ -39,6 +39,7 @@ script: |
   R CMD check plant*tar.gz --no-tests --no-manual 
   Rscript -e 'devtools::install();devtools::test()'
   pass=$?
+  Rscript -e 'plant::run_plant_benchmarks()'
```

Comment:
See the latest Canopy PR this code is all different - this version was a failed experiment not the final version.

Maybe we should merge Canopy first or you are going to have to merge and change all of this again later

Reply:
Yeah, i discovered that after sending this. Have subsequently added benchmarking onto your PR and reconciled these changes. see https://github.com/rafaqz/plant/tree/strategy_inheritance

So this branch won’t actually get merged, we’ll take the version on your strategy_inheritance branch